c10/util/complex_type.h

```diff
@@ -198,7 +198,7 @@ struct alignas(sizeof(T) * 2) complex_common {
   }
 
   template<typename U>
-  constexpr complex<T> &operator /=(const complex<U> &rhs) {
+  constexpr complex<T> &operator /=(const complex<U> &rhs) __ubsan_ignore_float_divide_by_zero__ {
```

Comment:
cc. @mruberry for the sanitization check and the failing macos test

Reply:
ubsan line LGTM; I'm not sure what's going on with the failing Mac test but I agree the issue looks legitimate. I think you'll have to debug it.